.github/workflows/test-go.yml

```diff
@@ -14,7 +14,11 @@ jobs:
           run: |
           sudo apt-get -y install
           btrfs-progs \
-          libgpgme-dev
+          libgpgme-dev \
```

Comment:
please align it with the changes that @ricardozanini is doing.

Reply:
This will be overridden by the other PR.